golang/pghoard_postgres_command_go.go

```diff
@@ -120,6 +120,16 @@ func restore_command(url string, output string) (int, error) {
 			}
 			output_path = path.Join(cwd, output)
 		}
+		// if file "<xlog>.pghoard.prefetch" exists, just move it to destination
+		xlogPrefetchPath := path.Join(path.Dir(output_path), xlog+".pghoard.prefetch")
+		_, err = os.Stat(xlogPrefetchPath)
```

Comment:
I wonder if possible to introduce the safety check whether the output_path is under PG data folder, which is similar check as the `webserver.py::_validate_target_path`. For example, in case escalated user from PG use the restore_command in this way:
```restore_command("url", "/root/.ssh/authorized_hosts", "../../tmp/malicious_file")``` Here `/tmp/malicious_file.pghoard.prefetch` file is prepared

Reply:
I added an extra check for xlog string:`^[A-F0-9]{24}$`

Reply:
Since only `postgres` user can execute `restore_command` and  if we make `postgres_restore_command_go` or `postgres_restore_command` file permission is 600 (`-rw-------  postgres postgres`, but it's irrelevant to the PR), above security checking maybe not much valuable.